kubernetes/cosmos/values.yaml

```diff
@@ -46,3 +46,4 @@ persistentVolume:
   accessModes:
     - ReadWriteMany
   size: 1Gi
+  readOnly: false
```

Comment:
@vrancurel do we want `readOnly: true` by default?

Reply:
Okay